test/Microsoft.SqlTools.ServiceLayer.UnitTests/QueryExecution/Execution/ServiceIntegrationTests.cs

```diff
@@ -431,15 +432,14 @@ public void GetSqlTextFromInvalidType()
                 .Complete();
             await queryService.HandleSimpleExecuteRequest(queryParams, efv.Object);
 
-            Query q;
-            queryService.ActiveQueries.TryGetValue(Constants.OwnerUri, out q);
-
-            // wait on the task to finish
+            Query q = queryService.ActiveQueries.Values.First();
```

Comment:
It would be nice to have a test that actually verifies we can run 2 queries against the same URI using this approach.